### PR TITLE
fix #55576 webcartop.jp

### LINKED ISSUE #55576: webcartop.jp
### Issue URL (Incorrect Blocking)
 [https://www.webcartop.jp/2020/05/527117/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.webcartop.jp%2F2020%2F05%2F527117%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
Platform: | And 10
AdGuard version: | 3.3.3 (231)
Browser: | Chrome
AdGuard mode: | VPN
Filtering quality: | High-quality
HTTPS filtering: | enabled
DNS requests blocking: | enabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Japanese

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#55576

### CODE REVIEW

**Comment by Yuki2718** [comment]
BTW a typo in the title (js/jp) :)

**Comment by Sergey-Lyapin** [comment]
> BTW a typo in the title (js/jp) :)

Thanks, fixed =)

File: EnglishFilter/sections/whitelist.txt [hunk 1]
```diff
@@ -214,6 +214,8 @@ copypastecharacter.com#$##contents { visibility: visible!important; }
 ! https://github.com/AdguardTeam/AdguardFilters/issues/55620
 @@||edge.microsoft.com/translate/auth
 @@||api.cognitive.microsofttranslator.com/translate?
+! https://github.com/AdguardTeam/AdguardFilters/issues/55576
```
Comment (Alex-302) [inline EnglishFilter/sections/whitelist.txt:217]: to JP filter, I guess

Comment (Sergey-Lyapin) [reply EnglishFilter/sections/whitelist.txt:217]: but `.single-ads` is from the base filter?

Comment (Yuki2718) [reply EnglishFilter/sections/whitelist.txt:217]: Or rather EasyList. @ryanbr

**Pull request merged**